test/test-tarantool/engine.cfg

```diff
@@ -0,0 +1,16 @@
+{
+    "set_language.test.lua": {
+        "memtx": {"engine": "memtx"}
+    },
+    "setopt_delimeter.test.lua": {
+        "memtx": {"engine": "memtx"}
+    },
+    "panic_on_broken_lsn.test.lua": {
+        "vinyl": {"engine": "vinyl"}
+    },
+    "manage_cluster.test.lua": {},
```

Comment:
Hm. There is no such test. Is it intentional?

Reply:
No, removed.